pkg/orchestration/wiring/wiringutil/knownshapes/asap_key.go

```diff
@@ -0,0 +1,43 @@
+package knownshapes
+
+import (
+	smith_v1 "github.com/atlassian/smith/pkg/apis/smith/v1"
+	"github.com/atlassian/voyager/pkg/orchestration/wiring/wiringplugin"
+)
+
+const (
+	ASAPKeyShapeName wiringplugin.ShapeName = "voyager.atl-paas.net/ASAPKey"
+)
+
+// +k8s:deepcopy-gen=true
+// +k8s:deepcopy-gen:interfaces=github.com/atlassian/voyager/pkg/orchestration/wiring/wiringplugin.Shape
+type ASAPKey struct {
+	wiringplugin.ShapeMeta `json:",inline"`
+}
+
+// NewASAPKey creates a new ASAPKey
+func NewASAPKey(_ smith_v1.ResourceName) *ASAPKey {
```

Comment:
Why is this argument still here? =) Why not remove it?

Reply:
👍 thanks, this was an oversight after I copy pasted